### DISCUSSION TITLE
MIDI CC commands constantly triggering and solution

### DISCUSSION THREAD

**Discussion by MalachyTarpey** [rsjaffe/MIDI2LR#734] [Problems]
ISSUE: After setting dials and sliders when using the "DEVELOP" tab in LR, midi controller constantly triggers LR commands that have been set until the MIDI2LR plugin crashes.

SOLUTION:  Two midi I/O cables were plugged into the midi controller keyboard.  The issue was resolved by unplugging the midi cables. ( midi controller keyboard is connected to the PC using a USB cable.)

**Comment by kinggump** [comment]
I also have this issue, as I connected the USB sound card. Can this issue be solved by the update of the software?

**Reply by MalachyTarpey** [reply]
Updating and downgrading midi2lr did not change anything for me. I use the LR classic version so can't upgrade LR further because I don't want to pay to upgrade again.

**Comment by antzpantz** [comment]
What MIDI controller are you using? It might be the issue of how the
controller is configured to send commands.

**Reply by MalachyTarpey** [reply]
The configuration wasn't the problem. The issue was resolved by unplugging unused midi cables.

**Comment by paul-cine** [comment]
The keyboard probably creates a midi feedback loop
